DESCRIPTION

```diff
@@ -33,8 +33,7 @@ Suggests:
     forcats,
     glue,
     here,
-    stringr,
-    dplyr
+    stringr
```

Comment:
stringr is a package that we don't really need. These are wrappers around base sub gsub and trimws

Reply:
Good point. I think I used it, but we can consider taking it out again, if we want to

Reply:
It only shows up under suggests, so I guess we can keep it here and then clean up description before the release?